netket/jax/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from netket.utils import HashablePartial
+
```

Comment:
`HashablePartial` can now be imported from both `nk.utils` and `nk.jax`. Why is this? (And which one is preferred?)